rampwf/score_types/detection/precision_recall.py

```diff
@@ -155,7 +155,13 @@ def mad_radius(y_true, y_pred, matches=None, iou_threshold=0.5,
     loc_true, loc_pred = _locate_matches(
         y_true, y_pred, matches, iou_threshold=iou_threshold)
 
-    return np.abs((loc_pred[:, 2] - loc_true[:, 2]) / loc_true[:, 2]).mean()
+    _, _, rad_true = loc_true.T
+    _, _, rad_pred = loc_pred.T
```

Comment:
Since this are actually arrays, instead of unpacking I would just index (`rad_true = loc_true[:, 2]`)

(but good clean-up to give it names!)

Reply:
Does it create a view instead of a copy ?

Reply:
I think both are views (the slicing certainly)